src/clr-angular/data/datagrid/datagrid-row.html

```diff
@@ -21,7 +21,10 @@
 -->
 <ng-template #detail>
   <div [id]="expandableId">
+    <div class="clr-sr-only">{{ commonStrings.keys.dategridExpandableRowsHelperText }}</div>
```

Comment:
I think this should be inside of the `clr-dg-row-detail` component instead of in its host.